test/src/main/java/org/apache/accumulo/test/functional/ManagerAssignmentIT.java

```diff
@@ -92,6 +93,8 @@
 
 public class ManagerAssignmentIT extends SharedMiniClusterBase {
 
+  private static AccumuloClient client;
```

Comment:
I think the diff below is large because of the variable name change from `c` to`client`. I understand why you changed the name, but it's hard to see if there are any other differences. I'm wondering if it makes sense to change the variable name back to `c` to review, and then rename right before merging.

Reply:
The diff is large because of whitespace differences. If you hide whitespace changes in GitHub UI, it's much smaller and much easier to read.